### PR TITLE
Initial structure for chains and assets

### PR DESCRIPTION
**Pull request by pedrorezende**
This PR initializes the repository with a basic json structure for Namada testnet chains and assets.

### CODE REVIEW

File: namada/assetList.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "namada",
+  "assets": [
+    {
+      "description": "The native token of Namada.",
+      "denom_units": [
+        {
+          "denom": "namnam",
```
Comment (brentstone) [inline namada/assetList.json:9]: probably better to express this formally as `unam`

File: namada/assetList.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "nam",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "namnam",
```
Comment (brentstone) [inline namada/assetList.json:18]: unam

File: namada/namada-housefire.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "chain_type": "namada",
+  "chain_id": "housefire-cotton.d3c912fee7462",
+  "slip44": 118,
+  "bech32_prefix": "nam",
+  "daemon_name": "nam",
+  "key_algos": ["secp256k1"],
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "namnam",
```
Comment (brentstone) [inline namada/namada-housefire.json:17]: unam

**Review by brentstone — APPROVED** [review]
Thx for starting this!

File: namada/namada-housefire.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/anoma/namada",
+    "language": {
+      "type": "rust",
+      "version": "1.21.4"
```
Comment (Fraccaman) [inline namada/namada-housefire.json:39]: 0.44.1

File: namada/namada-housefire.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "nam"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
```
Comment (Fraccaman) [inline namada/namada-housefire.json:32]: im not sure this is correct but i dont think it matters

File: namada/namada-internal-testnet.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/anoma/namada",
+    "language": {
+      "type": "rust",
+      "version": "1.21.4"
```
Comment (Fraccaman) [inline namada/namada-internal-testnet.json:39]: 0.44.1

File: namada/namada-internal-testnet.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  "chain_type": "namada",
+  "chain_id": "internal-devnet-44a.1bd3e6ca62",
+  "slip44": 118,
+  "bech32_prefix": "nam",
+  "daemon_name": "nam",
+  "key_algos": ["secp256k1"],
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "namnam",
```
Comment (Fraccaman) [inline namada/namada-internal-testnet.json:17]: unam

File: namada/namada-internal-testnet.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        "fixed_min_gas_price": 0.0025,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
```
Comment (Fraccaman) [inline namada/namada-internal-testnet.json:21]: lets leave it like this, we will change them later

**Review by Fraccaman — APPROVED** [review]

**Review by jurevans — APPROVED** [review]
Nice!

File: namada/namada-internal-testnet.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
+{
```
Comment (jurevans) [inline namada/namada-internal-testnet.json:1]: My only comment here is to maybe rename this config to `namada/namada-internal-devnet.json` to match the chain ID.

File: namada/namada-housefire.json (hunk 4 above)
Comment (pedrorezende) [reply namada/namada-housefire.json:39]: I think this states the Rust version. Which one should I use?

Comment (pedrorezende) [reply namada/namada-housefire.json]: I've removed this entry for now

**Pull request merged**